old/lib/LedgerSMB/oldHandler.pm

```diff
@@ -0,0 +1,209 @@
+######################################################################
+# LedgerSMB Accounting and ERP
+
+# http://www.ledgersmb.org/
+#
+# Copyright (C) 2006
+# This work contains copyrighted information from a number of sources all used
+# with permission.
+#
+# This file contains source code included with or based on SQL-Ledger which
+# is Copyright Dieter Simader and DWS Systems Inc. 2000-2005 and licensed
+# under the GNU General Public License version 2 or, at your option, any later
+# version.  For a full list including contact information of contributors,
+# maintainers, and copyright holders, see the CONTRIBUTORS file.
+#
+# Original Copyright Notice from SQL-Ledger 2.6.17 (before the fork):
+# Copyright (C) 2001
+#
+#  Author: Dieter Simader
+#   Email: dsimader@sql-ledger.org
+#     Web: http://www.sql-ledger.org
+#
+#  Contributors:
+#
+#
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#######################################################################
+#
+# this script is the frontend called from old/bin/$terminal/$script
+# all the accounting modules are linked to this script which in
+# turn execute the same script in old/bin/$terminal/
+#
+#######################################################################
+
+package lsmb_legacy;
+
+use LedgerSMB::User;
+use LedgerSMB::Form;
+use LedgerSMB::Locale;
+use LedgerSMB::App_State;
+use LedgerSMB::Middleware::SessionStorage;
+use LedgerSMB::Middleware::RequestID;
+use LedgerSMB::PSGI::Util;
+use LedgerSMB::Sysconfig;
+
+use Cookie::Baker;
+use Digest::MD5;
+use Log::Log4perl;
+use Try::Tiny;
+
+our $logger;
+
+
+sub handle {
+    # WARNING !!
+    #
+    # This function *must* be called in a forked process
+    #
+    # because it changes global state without clean-up
+    #
+    #
+    # Note that this function can receive a PSGI environment, but
+    # modifications aren't marshalled back to the fork()ing process!
+    my ($class, $psgi_env) = @_;
+
+    binmode (STDIN, ':utf8');
+    binmode (STDOUT, ':utf8');
+    binmode STDERR, ':utf8';
```

Comment:
Inconsistent use of brackets compared with preceding lines.

Reply:
You caught me copy/pasting from different sources! (thanks! :-) )

Reply:
Fix submitted.